webapp/channels/src/selectors/work_template.ts

```diff
@@ -3,14 +3,23 @@
 
 import {createSelector} from 'reselect';
 
-import {getFeatureFlagValue} from 'mattermost-redux/selectors/entities/general';
+import {getFeatureFlagValue, getLicense} from 'mattermost-redux/selectors/entities/general';
 import {GlobalState} from 'types/store';
+import {haveICurrentTeamPermission} from 'mattermost-redux/selectors/entities/roles';
+import {Permissions} from 'mattermost-redux/constants';
+import {isEnterpriseOrE20License} from 'utils/license_utils';
 
 export const areWorkTemplatesEnabled = createSelector(
     'areWorktemplatesEnabled',
     (state: GlobalState) => getFeatureFlagValue(state, 'WorkTemplate') === 'true',
-    (workTemplateFF) => {
-        return workTemplateFF;
+    (state: GlobalState) => getLicense(state),
+    (state: GlobalState) => haveICurrentTeamPermission(state, Permissions.CREATE_PUBLIC_CHANNEL) || haveICurrentTeamPermission(state, Permissions.CREATE_PRIVATE_CHANNEL),
+    (state: GlobalState) => haveICurrentTeamPermission(state, 'playbook_public_create'),
+    (state: GlobalState) => haveICurrentTeamPermission(state, 'playbook_private_create'),
```

Comment:
These keys are available from Permissions: `PLAYBOOK_PUBLIC_CREATE`, `PLAYBOOK_PRIVATE_CREATE`

Reply:
Ohhh I was not available when I made my branch, this probably appeared after my rebase!